loadgen/cmd/otelbench/main.go

```diff
@@ -147,15 +138,32 @@ func main() {
 		}
 	}
 
+	const statsFetcherDurationPadding = time.Second
+	from := time.Now().UTC()
 	for _, concurrency := range Config.ConcurrencyList {
 		for _, signal := range getSignals() {
 			for _, exporter := range getExporters() {
 				benchName := fullBenchmarkName(signal, exporter, concurrency)
-				result := runBench(fetcher, signal, exporter, concurrency)
+				result := runBench(ctx, signal, exporter, concurrency, func(b *testing.B) {
+					if fetcher == nil {
+						return
+					}
+					// after each run wait a bit to capture late metric arrivals
+					time.Sleep(statsFetcherDurationPadding)
+					to := time.Now().UTC()
+					stats, err := fetcher.FetchStats(ctx, from, to)
+					if err != nil {
+						fmt.Fprintf(os.Stderr, "error while fetching remote stats %s", err)
+						return
+					}
+					for unit, n := range stats {
+						b.ReportMetric(n, unit)
+					}
+					// advance the timestamp a bit to minimize the metrics influence of the previous run over the next run
+					from = to.Add(statsFetcherDurationPadding)
```

Comment:
does it also mean we'll miss the first `padding` of metrics from next bench run? I don't have a strong opinion here, just commenting as I'm not sure what value we get from adding 2nd padding here as we've already done `time.Sleep`.